core/modules/memman/MemManNone.h

```diff
@@ -44,6 +44,7 @@ class MemManNone : public MemMan {
 public:
 
     Handle lock(std::vector<TableInfo> const& tables, int chunk) override {
+               if (_alwaysLock) return true;
                (void)chunk;
```

Comment:
Returning true is wrong. It should be returning HandleType::ISEMPTY. I changed this and the other returns to HandleType values instead of integers.